service/lib/dinstaller/dbus/software/manager.rb

```diff
@@ -78,6 +81,8 @@ def initialize(backend, logger)
             backend.package_installed?(name)
           end
 
+          dbus_method(:UsedDiskSpace, "out SpaceSize:s") { backend.used_disk_space }
```

Comment:
Shouldn't this belong to the proposal iface?

Reply:
Moreover, I would offer more info, for example the number of packages to install.

Reply:
well, so far we agreed on that used space when we discuss it. I am sure in future we will need to have there more. About manager versus proposal, I am not sure where it belongs as I see reasons for both and having it in manager is easier for me for implementation in frontend as frontend so far does not use proposal.